.github/workflows/ci.yaml

```diff
@@ -23,14 +23,30 @@ jobs:
         go-version: "1.19"
     - name: Run tests
       run: make test
-  
+
+  check-codegen:
```

Comment:
I think that this should be in another workflow. We might still want to do the build even if we forgot to regenerate the manifests

Reply:
Because checking the code generation is a test, I think it has its place inside this workflow. However, currently the `build-and-push` job is run regardless of whether the tests have passed or not. I think we should add a condition on it (that the `unit-tests` job passes).

Reply:
ok, LGTM then :+1: